include/sharg/validators.hpp

```diff
@@ -431,8 +435,26 @@ class file_validator_base
                });
     }
 
+    std::string create_extensions_str()
+    {
+        std::string result{};
+
+        if (!extensions.empty())
+        {
+            result += "[";
+            for (std::string const & ext : extensions)
+                result += ext + ", ";
+            result.replace(result.size() - 2, 2, "]"); // repalce last ", " by "]"
+        }
+
+        return result;
```

Comment:
```suggestion
        if (extensions.empty())
            return "[]";

        std::string result{'['};
        for (std::string const & ext : extensions)
            result += ext + ", ";
        result.replace(result.size() - 2, 2, "]"); // replace last ", " by "]"
        return result;
```

I think it should return `[]` when there are no extensions?